lib/ansible/config/base.yml

```diff
@@ -193,10 +205,10 @@ COLLECTIONS_PATHS:
   description: >
     Colon separated paths in which Ansible will search for collections content.
     Collections must be in nested *subdirectories*, not directly in these directories.
-    For example, if ``COLLECTIONS_PATHS`` includes ``~/.ansible/collections``,
+    For example, if ``COLLECTIONS_PATHS`` includes ``${ANSIBLE_HOME}/collections``,
     and you want to add ``my.collection`` to that directory, it must be saved as
-    ``~/.ansible/collections/ansible_collections/my/collection``.
-  default: ~/.ansible/collections:/usr/share/ansible/collections
+    ``${ANSIBLE_HOME}/collections/ansible_collections/my/collection``.
```

Comment:
I would not document as an 'env var' as it might confuse the user on how this works

Reply:
fixed